cli/args/flags.rs

```diff
@@ -5685,6 +5690,26 @@ mod tests {
     );
   }
 
+  #[test]
+  fn short_permission_flags() {
+    let r = flags_from_vec(svec!["deno", "run", "-RW", "gist.ts"]);
```

Comment:
Shouldn’t others be tested as well?

Reply:
Sure, do you want to open a PR that exercises all of them?